README.md

```diff
@@ -214,7 +251,6 @@ Note that this has been not yet implemented for pull requests i.e. `pr` subcomma
 
 ## Roadmap
 
-- [ ] trigger ofBorg builds (write @GrahamcOfBorg build foo into pull request discussion)
```

Comment:
Per comment from @grahamc, commenting explicitly for the OfBorg bot is now unnecessary, so I've removed this item from the roadmap.